src/components/Buttons/IconButton/IconButton.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import './IconButton.scss';
+
+class IconButton extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <div
+                className={'icon-button ' + this.props.class}
+                style={{ width: this.props.width }}
+                onClick={this.props.clickEvent}>
+                {this.props.position === 'left' ? (
+                    <>
```

Comment:
Why?

Reply:
Otherwise it would complain about closing div